src/skin/legacy/legacyskinparser.cpp

```diff
@@ -1049,6 +1049,13 @@ QWidget* LegacySkinParser::parseOverview(const QDomElement& node) {
             &PlayerManager::slotLoadLocationToPlayerMaybePlay);
     connect(overviewWidget, &WOverview::cloneDeck,
             m_pPlayerManager, &PlayerManager::slotCloneDeck);
+    // This signal stems from AnalysisFeature, and the overview can now
+    // render the analysis progress like when listening to Playermanager's
+    // analysis progress signal when loading a track.
+    connect(m_pLibrary,
+            &Library::onTrackAnalyzerProgress,
+            overviewWidget,
+            &WOverview::onTrackAnalyzerProgress);
```

Comment:
Here we have probably the a queued connection. Without the Qt::DirectConnections before, we would have the queued connection at the place wher it enteres a main thread object and here we would have automatically a direct connection. 
It think this is the way to go.

Reply:
Thanks for the hint!
The connection are added in #14150, and since that (probably) will be merged before this PR I'll fix it there.